Assets/UI/launchbar_CQUI.lua

```diff
@@ -0,0 +1,250 @@
+-- ===========================================================================
+-- Base File
+-- ===========================================================================
+include("LaunchBar");
+
+-- ===========================================================================
+-- Cached Base Functions
+-- ===========================================================================
+BASE_CQUI_OnOpen = OnOpen;
+BASE_CQUI_Unsubscribe = Unsubscribe;
+BASE_CQUI_Subscribe = Subscribe;
+BASE_CQUI_Initialize = Initialize;
+
+-- ===========================================================================
+-- CQUI Members
+-- ===========================================================================
+-- Launchbar Extras. Contains the callback and the button text
+local m_LaunchbarExtras:table = {};
+
+-- ===========================================================================
+-- CQUI Extension Functions
+-- These functions replace the unmodifed versions
+-- ===========================================================================
+function OnOpen()
+  local screenX, screenY:number = UIManager:GetScreenSizeVal();
+  if screenY <= 850 then
+    Controls.ScienceHookWithMeter:SetOffsetY(-5);
+    Controls.CultureHookWithMeter:SetOffsetY(-5);
+  end
+
+  BASE_CQUI_OnOpen();
+end
+
+-- ===========================================================================
+function Unsubscribe()
+  BASE_CQUI_Unsubscribe();
+  -- Vaniila-only function, because OnInputActionTriggered may be overriden elsewhere (like XP1)
+  Events.InputActionTriggered.Remove( OnInputActionTriggeredVanilla );
+end
+
+-- ===========================================================================
+function Subscribe()
+  BASE_CQUI_Subscribe();
+  -- Undo the default for this Event
+  Events.InputActionTriggered.Remove( OnInputActionTriggered );
+  -- Vaniila-only function, because OnInputActionTriggered may be overriden elsewhere (like XP1)
+  Events.InputActionTriggered.Add( OnInputActionTriggeredVanilla );
+end
+
+-- ===========================================================================
+-- CQUI: Unique Functions
+-- These functions are unique to CQUI, do not exist in the Base WorldInput.lua
+-- ===========================================================================
+function OnInputActionTriggeredVanilla( actionId )
+  -- Vanilla-only function, because OnInputActionTriggered may be overriden elsewhere (like XP1)
+  OnInputActionTriggered( actionId )
+end
```

Comment:
Are you sure we need these functions? I was looking at the diff between the last version by Azurency (Hash: b73a5fd4) and the base game files that it is supposed to match (Hash: 44400580). It looked like the line with `Events.InputActionTriggered` and even the comment above it was from the unmodded version. In the version after the last Azurency modifications (Hash: 8f28fee5), the matching lines were marked as being CQUI customizations.

Reply:
Not sure that we need them, no... was just copying over the differences between the two files and this was marked as a difference.  Will remove and test.  I plan to play a game in a bit as I haven't actually played a game in a week, just been starting and playing a few turns to test changes :)

Reply:
removed them.  updated PR with that change.